trusted-firmware-m/platform/ext/common/tfm_platform.c

```diff
@@ -13,7 +13,11 @@
 __WEAK enum tfm_hal_status_t tfm_hal_platform_init(void)
 {
     __enable_irq();
+
+    /* Only if UART1 is used by TF-M do we initialize it. */
+#ifdef SECURE_UART1
```

Comment:
How will this impact boards that are having `SECURE_UART1` to off ?

For example the Zephyr board `mps2_an521_ns` which is frequently been used for testing.

Reply:
I am aware of any methods to test the code with external boards, but let me know if you are aware of any.

Reply:
The current solution does not impact any other boards.